File: rockpaperscissors/src/main/java/Player.java
public class Player {
    private final String name;

    public Player(String name) {
        this.name = name;
    }

    public Gesture throwRock() {
        return Gesture.ROCK;
    }

    public Gesture throwScissors() {
        return Gesture.SCISSORS;
    }

    public Gesture throwPaper() {
        return Gesture.PAPER;
    }

    public String getName() {
        return name;
    }
}

File: rockpaperscissors_py/main/Player.py
from main import GestureEnum as Gesture

class Player:
    def __init__(self, name):
        self.name = name

    def throw_rock(self):
        return Gesture.ROCK

    def throw_scissors(self):
        return Gesture.SCISSORS

    def throw_paper(self):
        return Gesture.PAPER

    def get_name(self):
        return self.name
